### DISCUSSION TITLE
Chat in footer of saleor.io goes to spectrum instead of here

### DISCUSSION THREAD

**Discussion by jurjendevries** [saleor/saleor#9122] [General] [answered]
In the footer of saleor.io is a link to a chat of Spectrum, but the Spectrum page refers to Github discussions. I would suggest to update the icon with Github discussions icon and link it directly to https://github.com/saleor/saleor/discussions

**Comment by karolkielecki** [comment, marked as answer]
Hi @jurjendevries. Thanks for reporting! I've just passed it to the team. Cheers!

**Comment by jurjendevries** [comment]
Thanks. Would be nice if they update it.
